pkgs/servers/sql/postgresql/generic.nix

```diff
@@ -114,7 +115,8 @@ let
       pkg-config
       removeReferencesTo
     ]
-      ++ lib.optionals jitSupport [ llvmPackages.llvm.dev nukeReferences ];
+      ++ lib.optionals jitSupport [ llvmPackages.llvm.dev nukeReferences ]
+      ++ lib.optionals (atLeast "17") [ bison flex perl docbook_xml_dtd_45 docbook-xsl-nons libxslt ];
```

Comment:
See https://peter.eisentraut.org/blog/2024/08/13/the-new-postgresql-17-make-dist for some more details